test/integration/data/utt-report-details.stdout

```diff
@@ -0,0 +1,26 @@
+
+------ Monday, Aug 20, 2018 (week 34) to Tuesday, Aug 21, 2018 (week 34) -------
+
+Working Time: 0h24
+Break   Time: 0h00
+
+----------------------------------- Projects -----------------------------------
+
+(0h12) project_1: task_1, task_2
+(0h12) project_2: task_1, task_2
+
+---------------------------------- Activities ----------------------------------
+
+(0h06) project_1: task_1
+(0h06) project_1: task_2
+(0h06) project_2: task_1
+(0h06) project_2: task_2
+
+
+----------------------------------- Details ------------------------------------
+
+(0h06) 08:00-08:06 project_1: task_1
+(0h06) 08:06-08:12 project_2: task_1
+(0h06) 08:12-08:18 project_1: task_2
+(0h06) 08:18-08:24 project_2: task_2
```

Comment:
What I find confusing is that we asked for a multi-day reports, but it seems to show only the details for a single day. Shouldn't it display for all the days requested?

Also, I probably missed something, but why don't we see in this section and the other sections `task_3`?

Reply:
Thanks for catching this.  I messed up when I wrote the test spec -- it was run with `--now` one day too early.  And then I didn't catch that the output should have specified several days of details.

Test fixed in eab56f0

(I must have been sleepy or exhausted when I did this... sorry about that.)